File: Day5/C/Ques18.c
//Strong Number
#include<stdio.h>
int main(){
    int n,num,a,temp=0;
    printf("Enter a Number:");
    scanf("%d",&n);
    num = n;
    while(n>0){
        a=n%10;
        int fact=1;
        for(int i=1;i<=a;i++){
           fact =fact*i;
        }
    temp=temp+fact;
    n=n/10;
     }
     if(temp==num){
        printf("%d is a Strong Number",num);
     }
     else{
        printf("%d is a not a Strong Number",num);
     }
     return 0;

}

File: Day5/Python/Ques18.py
a=int(input("Enter a number:"))
temp=a
sum=0
while temp>0:
    fact=1
    b=temp%10
    for i in range(1,b+1):
        fact = fact*i
    sum+= fact
    temp=temp//10
if sum==a:
    print(a," is a Strong Number.")
else:
    print(a,"is not a Strong Number.")
    